modules/ROOT/pages/dashboard-custom-config-graph.adoc

```diff
@@ -138,71 +140,73 @@ TODO_ABOVE:
 ////
 
 === Legends (Advanced Setting)
-In the Advanced configuration tab, you can configure legends for your graphs.
+
+In the *Advanced configuration* tab, configure legends for your graphs.
 
 image::graph-advanced-legend.png[Advanced Graph Settings: Legend]
 
 [%header,cols="1,4"]
 |===
 2+| Legend Options
 | Show  | Unchecking hides the legend. Checked by default.
-| As Table | Checking displays the selected legend Values (such as, Min and Max) in a table. Unchecked by default.
-| To the Right | Checking display the legend to the right side. Unchecked by default.
-| Width | Available when To the Right is checked. Sets the minimum width (in pixels) for the legend. You can leave it blank to allow for the automatic width selection.
+| As Table | Check to display the selected legend Values (such as, Min and Max) in a table. Unchecked by default.
+| To the Right | Check to display the legend to the right side. Unchecked by default.
```

Comment:
display the legend on the right side is more natural in the explanation. Where does it display if this is unchecked. Also, "Unchecked by default" isn't a sentence. Add a column for Default Value or Default State to this table?